scalafix-rules/src/main/scala/scalafix/internal/rule/CompilerTypePrinter.scala

```diff
@@ -222,6 +225,7 @@ class CompilerTypePrinter(g: ScalafixGlobal, config: ExplicitResultTypesConfig)(
 
   private def isPossibleSyntheticParent(tpe: Type): Boolean = {
     definitions.isPossibleSyntheticParent(tpe.typeSymbol) ||
+    CompilerCompat.serializableClass(g).toSet[Symbol](tpe.typeSymbol) ||
```

Comment:
isn't that redundant with the check above?

Reply:
not really... for scala 2.12 and 2.11, this method `definitions.isPossibleSyntheticParent` wasn't checking both Serializable traits.

Reply:
got it, thanks!